client/lib/products-values/translations.js

```diff
@@ -28,6 +28,35 @@ export const getJetpackProductsShortNames = () => {
 	};
 };
 
+export function getShortNameCallbackForJetpackSearch(
+	productObject,
+	tierKey = 'price_tier_slug',
+	usageKey = 'price_tier_usage_quantity'
+) {
+	switch ( productObject[ tierKey ] ) {
+		case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_100_RECORDS:
+			return translate( 'Up to 100 records' );
+		case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_1K_RECORDS:
+			return translate( 'Up to 1,000 records' );
+		case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_10K_RECORDS:
+			return translate( 'Up to 10,000 records' );
+		case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_100K_RECORDS:
+			return translate( 'Up to 100,000 records' );
+		case CONSTANTS.JETPACK_SEARCH_TIER_UP_TO_1M_RECORDS:
+			return translate( 'Up to 1,000,000 records' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Tier 5: Up to 1,000,000 records' )` *ES Score: 9*